Doppler.BigQueryMicroservice/Repository/Classes/UserAccessByUserRepository.cs

```diff
@@ -97,10 +97,12 @@ public async Task<bool> MergeEmailsAsync(int userId, List<string> emails)
     )
     THEN
     DELETE
-    ;";
-
-                    connection.Execute(sql, new { Emails = dt, IdUser = userId });
-                    return true;
+    OUTPUT
+        $action as Action,
+        inserted.Email,
+        deleted.Email;";
```

Comment:
Nice! you found a great way to get information from the merge ❤️

I have a question. As I understand, it returns 3 columns: `action` (with the action), _unamed column 1_ (with the inserted email or null), _unamed column 2_ (with the deleted email or null).

How is this being mapped to a `MergeResponse` object?

```csharp
    public class MergeResponse
    {
        public string Action { get; set; }
        public string Email { get; set; }
    }
```